Update examples

kuba-- opened the pull request:
Signed-off-by: kuba-- <kuba@sourced.tech>
Closes https://github.com/src-d/go-mysql-server/issues/597

erizocosmico commented on SUPPORTED_CLIENTS.md:
```diff
@@ -28,14 +28,14 @@ These are the clients we actively test against to check are compatible with go-m
 try:
     with connection.cursor() as cursor:
-        sql = "SELECT foo FROM bar"
+        sql = "SELECT * FROM commit_files LIMIT 1"
```
These are go-mysql-server examples, not gitbase examples. I don't think we should mix the two of them

kuba-- commented:
@smola - WDYT? Maybe docs should follow the schema from _example

kuba-- commented:
@src-d/data-processing, @smola 
I replaced all `gitbase` related terms by `mydb` and `mytable`.
Also I've added C-test and C example. I know it's a language for dinosaurs, but I'm one of them ;)
